### ISSUE TITLE
Licensing changes explained

### ISSUE THREAD

**Issue by AbandonedCart** [HiddenRamblings/TagMo#497]
TagMo is not going to be a paid app. It is not going to be privately sourced. It is not going to change from what it is today. The purpose of updating and clarifying the secondary license is much more wholesome than that.
It is to protect the current state of the proprietary code that makes some of the more "advanced" features from other parties selling it. It is to prevent the Apple developers from throwing the code into an app and charging $10 after 3 uses to keep using it. 
Put simply, it is to prevent anyone from trying to exploit the code for profit. As I am the author, license holder, and in good standing as a Google Play developer, that also means having legal recourse for enforcement action backed by the developer program.
That said, the license terms are not unrealistic. It is as simple as open source code or visible credit to TagMo for writing it. Nobody should pay for something under false pretenses.
